packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -564,6 +573,112 @@ describe( 'Navigation', () => {
 		expect( quickInserter ).toBeTruthy();
 	} );
 
+	it( 'supports navigation blocks that have inner blocks within their markup and converts them to wp_navigation posts', async () => {
```

Comment:
Can we add another test like this one where inner blocks are created in code view and then we allow them to be converted to an entity and _then_ try and Create a new Navigation using `Select Menu -> Create New`.

Prior to my fix that test should break. I think we need that coverage.